tests/python/pants_test/backend/python/tasks/BUILD

```diff
@@ -177,7 +177,7 @@ python_tests(
         "testprojects/src/python:python_distribution_directory",
     ],
     tags={"integration", "partially_type_checked"},
-    timeout=240,
+    timeout=360,
```

Comment:
I think the ipex tests were causing the timeout because we didn't bump the timeout when adding them.